package.json

```diff
@@ -34,9 +34,9 @@
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
-    "url": "https://github.com/creditkarma/thrift-server"
+    "url": "https://github.com/StatesTitle/thrift-server"
   },
   "devDependencies": {
-    "lerna": "^3.14.1"
+    "lerna": "^4.0.0"
```

Comment:
I'm guessing no breaking changes we care about

Reply:
Here's the changelog: https://github.com/lerna/lerna/blob/main/CHANGELOG.md

It didn't seem like anything that I was too worried about, a lot of exports but I don't think this code uses `lerna` programmatically, and the only other thing was dropping Node 6 and 8 support.